custom/icds_reports/data_pull/data_pulls.py

```diff
@@ -74,12 +110,12 @@ def __init__(self, db_alias, *args, **kwargs):
     def get_queries(self):
         return [query_class(self.location_id, self.month).sql_query for query_class in self.queries]
 
-    def run(self):
+    def _get_data_files(self):
         result = {}
         for query_class in self.queries:
             query_obj = query_class(self.location_id, self.month)
             result[query_obj.result_file_name] = query_obj.run(self.db_alias)
-        return result
+        return self.post_run(result)
```

Comment:
I don't think this should be called here since it's called in `run`

Reply:
aha! thanks for catching that, updated.